README.md

```diff
@@ -34,7 +34,10 @@ custom:
   logForwarding:
     destinationARN: '[ARN of Lambda Function to forward logs to]'
     # optional:
-    filterPattern: `[filter pattern for logs that are sent to Lambda function]'
+    filterPattern: '[filter pattern for logs that are sent to Lambda function]'
+    stages:
```

Comment:
Is it intentional that `stages` be optional and the default be all stages? If so, it might be helpful to point that out in the README.

Reply:
yeah, its optional. there was a `#optional` on top so i thoguht it was an indication that all params above that comment is optional.

Reply:
Oh, I see. LGTM then. :)